front/app/containers/Admin/pagesAndMenu/containers/ProjectsList/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+// components
+import SectionFormWrapper from '../../components/SectionFormWrapper';
+import ShownOnPageBadge from '../../components/ShownOnPageBadge';
+import ViewCustomPageButton from '../CustomPages/Edit/ViewCustomPageButton';
+import ProjectsListContent from './ProjectsListContent';
+import HelmetIntl from 'components/HelmetIntl';
+
+// hooks
+import useLocalize from 'hooks/useLocalize';
+import { useIntl } from 'utils/cl-intl';
+import { useParams } from 'react-router-dom';
+import useCustomPage from 'hooks/useCustomPage';
+
+// utils
+import { pagesAndMenuBreadcrumb } from '../../breadcrumbs';
+import { adminCustomPageContentPath } from '../../routes';
+import { isNilOrError } from 'utils/helperUtils';
+
+// i18n
+import messages from './messages';
+
+const ProjectList = () => {
+  const localize = useLocalize();
+  const { formatMessage } = useIntl();
+
+  const { customPageId } = useParams() as { customPageId: string };
+  const customPage = useCustomPage({ customPageId });
+
+  if (isNilOrError(customPage)) {
+    return null;
+  }
+
+  const areaIds = customPage.relationships.areas.data.map((area) => area.id);
+  const topicIds = customPage.relationships.topics.data.map(
+    (topic) => topic.id
+  );
```

Comment:
You could move this to ProjectListContent because it's only needed in that component.